app/js/actions/pastOrders.js

```diff
@@ -17,18 +17,18 @@ export const getPastOrdersFailure = error => ({
 });
 
 export const getPastOrders = (patientUuid, careSetting) => (dispatch) => {
-  dispatch(loading(true));
+  dispatch(loading('LOAD_PAST_ORDERS', true));
   return axios.get(`${apiBaseUrl}/order?careSetting=${careSetting}&patient=${patientUuid}&status=inactive&t=drugorder&v=full`)
```

Comment:
I'll suggest that you create an axios instance with a base URL in some helper file then export it to get rid of appending ${apiBaseUrl} every time you make a call

Reply:
@kwahalf  Thanks for the suggestion. 
Have noted it.
Your feedback would be implemented as a separate task as this current PR is for implementation of the generic loader.

Reply:
@kwahalf Thank you for the idea. This requires a change in all our previous tasks. A new ticket has been created for that [here](https://issues.openmrs.org/browse/OEUI-91)